README.md

```diff
@@ -262,4 +262,8 @@ and run that simulator as usual:
         $ cd  builds/RV32CIUxCHERI_<CPU>_bluesim
         $ make compile simulator test isa_tests
 
+#### Clone submodules
+Use the following command to update submodule after cloneing the repostiory: 
+
+$ git submodule update --init --recursive
```

Comment:
Looking at the github preview, it looks like this isn't formatted right (too big and bold). Is it possible it should be indented to be more like other example commands?

Reply:
Format it as similar to other commands.